docs/contents.md

```diff
@@ -10,9 +10,12 @@ The **SOC Utilities** solution pack contains the following resources:
 |08 - SOC Utilities |
 |:---------------------------------------------|
 
-| Playbook Name                                                                | Description                                                                                                                                     |
-|:-----------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------|
-| Clone FortiSOAR Module                                       | Playbook allows to clone a FortiSOAR module. |
+| Playbook Name                                        | Description                                                                                                                   |
+|:-----------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------|
+| Clone FortiSOAR Module                               | Playbook allows to clone a FortiSOAR module.                                                                                  |
+| Terminate Pending Manual Inputs                      | Terminates all manual input pending for user action/inputs that have been created before the specified timestamp.             |
+| Terminate Awaiting Data Ingestion playbooks          | Terminates list of data ingestion playbooks which are stuck in awaiting state from last x minutes                             |                                                                      |
```

Comment:
Terminates data ingestion playbooks which are stuck in awaiting state from last x minutes